docs/fundamentals/diagnostics/runtime-events.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Runtime events"
+description: Review diagnostic events emitted by .NET runtime (CoreCLR) that can be used with ETW, LTTng, or EventPipe.
+ms.date: "11/13/2020"
+ms.topic: reference
+helpviewer_keywords: 
+  - "runtime events (CoreCLR)"
+  - "ETW, EventPipe runtime events (CoreCLR)"
+---
+
+# .NET runtime events
+
+The .NET runtime (CoreCLR) emits various events that can be used to diagnose issues with your .NET application that can be consumed via various mechanisms such as `ETW`, `LTTng`, and `EventPipe`.
+
+This document serves as a reference on the events that are fired by .NET Core runtime.
+
+For runtime events in .NET Framework, please refer to the [CLR ETW Events document](../../framework/performance/clr-etw-events.md).
```

Comment:
"please refer" is a bit formal. We also use the word article over document/topic. But I don't think that is really needed.

```suggestion
For runtime events in .NET Framework, see [CLR ETW Events](../../framework/performance/clr-etw-events.md).
```